app/helpers/application_helper.rb

```diff
@@ -76,6 +76,10 @@ def tin_options_for_select(include_ssn_no_employment: false, include_atin: false
     options
   end
 
+  def months_in_home_options_for_select
+    {I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.less_than_six") => 6, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.six_to_seven") => 7, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.seven_to_eight") => 8, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.eight_to_nine") => 9, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.nine_to_ten") => 10, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.ten_to_eleven") => 11, I18n.t("views.ctc.questions.dependents.child_residence.reveal.select_options.eleven_to_twelve") => 12}
```

Comment:
Yes! I was a little conflicted about this. I know we talked about doing string enums for this but I went with `months_in_home`because that is a column that was already in the dependents' model and was used in this form app/views/dependents/_form.html.erb. 

In that form, they ask "How many months did they live in your home in %{year}?" and give a drop down of numbers from 0-12. So I guess in that case, less than six would be 5. If we wanted to use enums, we could but then we would have to backfill all the months_in_home values into their enum values. Or I guess we could make a new column and backfill values from months_in_home and lived_with_more_than_six_months into it. And then later drop those columns if it's okay with Max. 

What do you think, is it worth it to do a bunch of rake tasks to change it to enums that might be a bit more syntactically clear or to change the logic on the present months_in_home to check for >= 5 months? @caitbot